README.md

```diff
@@ -6,14 +6,14 @@ This tool aims to ease the micro-ROS integration in a STM32CubeMX project.
 
 1. In the `root` folder, generate your STM32CubeMX project. A sample project can be generated with the provided `sample_project.ioc`.
 2. Make sure that your STM32CubeMX project is using a `Makefile` toolchain under `Project Manager -> Project`
-3. Make sure that if you are using FreeRTOS, the micro-ROS task has more than 10 kB of stack.
+3. Make sure that if you are using FreeRTOS, the micro-ROS task **has more than 10 kB of stack**.
 4. To use the default UART transport based on DMA:
```

Comment:
Please change this and add another section below that is called "transport configuration". In this step 4 we should refer to this new section.

The new section must have subsections explaining how to configure the transport in CubeMX depending on if it is a serial with DMA, serial with interrupt and so on